### DISCUSSION TITLE
CAN I USE DRILL FEAUTRE?

### DISCUSSION THREAD

**Discussion by rindy8707** [apache/superset#23557] [General]
superset version: 2.0.1
I want to use the "drill by" feature. SO I write "DRILL_TO_DETAIL"=True to superset_config.py.
But, It is not Working. 

How can I use the "drill by" feature?

**Comment by mdeshmu** [comment]
Drill By is not released yet.

Drill to Detail is not same as Drill By. 

You can read more here: https://preset.io/blog/drill-down-and-drill-by/

Please post same question only once.
